lib/reference/keywordSets.js

```diff
@@ -182,6 +182,7 @@ keywordSets.vendorSpecificPseudoElements = new Set([
 	'-moz-range-progress',
 	'-moz-range-thumb',
 	'-moz-range-track',
+	'-moz-focus-inner',
```

Comment:
[trivial] This list seems to be ordered alphabetically, so it may be more readable to put `-moz-focus-inner` before `-moz-progress-bar`. (ignorable)